packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -145,7 +145,11 @@
             </card>
         </div>
 
-        <error-page v-else-if="shouldShowErrorPage" />
+        <error-page
+            v-else-if="shouldShowErrorPage"
+            :header="$t(`errorMessages.${getCheckoutError}.heading`)"
```

Comment:
What do you think about passing instead passing the checkout error type as a prop and building up the translation in the error page component instead?